scout/parse/models.py

```diff
@@ -1,450 +0,0 @@
-"""Class to hold information about scout load config and subclasses used"""
```

Comment:
I moved this file from here to the models, under models/case. I think this is its place instead!